### PR TITLE
cisco_nxos_show_ip_bgp_summary_vrf update to support 32-bits ASN outputs

### PR DESCRIPTION
**Pull request by burningnode**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
cisco_nxos_show_ip_bgp_summary_vrf.textfsm, cisco_nxos, show_ip_bgp_summary_vrf

##### SUMMARY
I'd like to update and separate this template because it fails to parse the output with 32-bit ASNs and line breaks/spaces relative to this output. I also added a separate entry for this template/command in the index.

### CODE REVIEW

File: templates/cisco_nxos_show_ip_bgp_summary_vrf.textfsm [hunk 1]
```diff
@@ -6,6 +6,20 @@ Value Required NEIGH_AS (\d+)
...
+  ^(\s+\d+?){5}\s+\S+?\s+${STATE_PFXRCD}\s*$$ -> Record
+  ^${BGP_NEIGH}\s+\S+\s+${NEIGH_AS}(\s+\d+?){5}\s+\S+?\s+${STATE_PFXRCD}\s*$$ -> Record
+  # Match other text lines
+  ^.+
+  # Match lines that are spaces
+  ^\s+$$
+  # Errors
+  ^.+ -> Error
+
+Done
```
Comment (jmcgill298) [inline templates/cisco_nxos_show_ip_bgp_summary_vrf.textfsm:24]: I don't see this state being used

File: templates/cisco_nxos_show_ip_bgp_summary_vrf.textfsm [hunk 2]
```diff
@@ -6,6 +6,20 @@ Value Required NEIGH_AS (\d+)
...
   ^BGP summary information for VRF ${VRF}, address family IPv4 Unicast
+  # Match RID and Local AS
   ^BGP router identifier ${ROUTER_ID}, local AS number ${LOCAL_AS}
-  ^${BGP_NEIGH}\s+\S+\s+${NEIGH_AS}(\s+\d+?){5}\s+\S+?\s+${STATE_PFXRCD}$$ -> Record
+  # Match Neighbor lines
+  ^${BGP_NEIGH}\s+\S+\s+${NEIGH_AS}.* -> Continue
+  ^(\s+\d+?){5}\s+\S+?\s+${STATE_PFXRCD}\s*$$ -> Record
+  ^${BGP_NEIGH}\s+\S+\s+${NEIGH_AS}(\s+\d+?){5}\s+\S+?\s+${STATE_PFXRCD}\s*$$ -> Record
+  # Match other text lines
+  ^.+
```
Comment (jmcgill298) [inline templates/cisco_nxos_show_ip_bgp_summary_vrf.textfsm:18]:
```suggestion
```
This conflicts with the reasons for having the following 2 matches

**Review by jmcgill298 — CHANGES_REQUESTED** [review]
Thanks @burningnode 

I had a couple of comments. Also, we would like the new test cases to be added to the directory instead of replacing the existing test cases.

**Comment by burningnode** [comment]
Hi @jmcgill298 

Thanks for reviewing this PR. I have been tasked with other things in the meantime and was not closely following up on this.

Normally I have adjusted to have the template and moved into its own directory. I forgot to rebase everything to make this looks cleaner, but it should be in 

templates/cisco_nxos_show_ip_bgp_summary_vrf.textfsm 
tests/cisco_nxos/show_ip_bgp_summary_vrf/cisco_nxos_show_ip_bgp_summary_vrf.raw
tests/cisco_nxos/show_ip_bgp_summary_vrf/cisco_nxos_show_ip_bgp_summary_vrf.yml

I removed the wrong one linked to 'show_ip_bgp_summary'
tests/cisco_nxos/show_ip_bgp_summary/cisco_nxos_show_ip_bgp_summary_vrf.yml

I have adjusted my code taking your comments into account and I agree I may have not properly used the outlined statements. I tested it on my devices, output looks OK.

Regarding your CI tests, I'm not sure to completely understand the error here 

```
----------------------------- Captured stdout call -----------------------------
Error on line: ['cisco_nxos_show_ip_bgp_summary_vrf.textfsm', ' .*', ' cisco_nxos', ' sh[[ow]] ip b[[gp]] s[[ummary]] v[[rf]]']
Error Message: Current Command len '31' larger then previous '16', for command 'show_ip_bgp_summary_vrf.textfsm'
=========================== short test summary info ============================
FAILED tests/test_index_order.py::test_index_ordering - assert False
======================= 1 failed, 1077 passed in 22.21s ========================
ERROR: InvocationError for command /home/travis/build/networktocode/ntc-templates/.tox/py36/bin/pytest -vv (exited with code 1)
```

Could you point me in the proper direction so I can make this pass,

Thank you very much,

**Comment by jmcgill298** [comment]
@burningnode it is complaining because the index file does not have the entries in the order specified.

https://github.com/networktocode/ntc-templates/blob/master/templates/index#L6-L10

**Comment by burningnode** [comment]
@jmcgill298 thank you. I understood, index was failing but I completely missed the rules at the top.

Modified a few things to parse more values and pass the checks.
Looks ok now.

**Review by jmcgill298 — APPROVED** [review]

**Comment by jmcgill298** [comment]
Thanks @burningnode

**Pull request merged**